docker/Dockerfile

```diff
@@ -274,12 +274,13 @@ CMD ["bash"]
 ############ build #############################################################
 FROM dev AS build
 
+ARG CUSTOM_CMAKE_ARGS="-DCMAKE_BUILD_TYPE=Release"
 # build ROS workspace
 RUN if [[ -x "$(command -v colcon)" ]]; then \
         source /opt/ros/${ROS_DISTRO}/setup.bash && \
         [[ -f /opt/ws_rmw_zenoh/install/setup.bash ]] && source /opt/ws_rmw_zenoh/install/setup.bash ; \
         [[ -f /opt/ws_base_image/install/setup.bash ]] && source /opt/ws_base_image/install/setup.bash ; \
-        colcon build --cmake-args -DCMAKE_BUILD_TYPE=Release ; \
+        colcon build --cmake-args ${CUSTOM_CMAKE_ARGS} ; \
```

Comment:
What happens if set to empty string? Still valid?

Reply:
Tested in minimal examples -> still valid